### ISSUE TITLE
v0.5.0 was a breaking change in at least 1 scenario

### ISSUE THREAD

**Issue by eriktrom** [html-next/ember-gestures#112]
I released 0.5.0 then asked a buddy to install it. His app uses 2.12 and ember-cli 2.18. the `@ember/*` imports within the instance initializer broke his app. (the cascade was heavy from there)

I re-published 0.4.8 as 0.6.0 and made a full sem ver bump to 1.0.0 for the #111 

No one should be effected unless they purposely make a major sem ver bump, or install this from scratch into an old app.

That said, *I don't really know why his app broke in this way given we use ember-try w/ 2.12 and it passes when visiting the index route. Unless our tests are completely broken.* I've filed an issue referencing this one to investigate that further.

In short, i'm making this issue to notify the world that 1.0.0 may or may not break your ember 2.x app. If `@ember/*` is working in your app, it won't. If not, it will.

Use 0.6.0 if this is the case.
